solutions/cf-231A.mdx

```diff
@@ -7,7 +7,7 @@ author: Sathvik Chundru
 
 [Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
 
-## Solution (Greedy)
+## Solution
 
 Let's consider the following test case:
 
```

Comment:
i don't know if we need this (the test case), what do other people think?

Reply:
i think remove it

Reply:
removed